src/platform/packages/shared/kbn-background-search/src/components/background_search_restored_callout.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import React from 'react';
+import useObservable from 'react-use/lib/useObservable';
+import { type Observable, distinctUntilChanged, of, scan } from 'rxjs';
+import { css } from '@emotion/react';
+import { EuiCallOut, EuiText, EuiPanel } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { SearchSessionState } from '@kbn/data-plugin/public';
+import { getBackgroundSearchState$, isBackgroundSearchEnabled } from '../services';
+
+const STICKY_STATES = new Set<SearchSessionState>([
+  SearchSessionState.Restored,
+  SearchSessionState.BackgroundLoading,
+]);
+
+export function BackgroundSearchRestoredCallout() {
+  const state$ = getBackgroundSearchState$();
+
+  const show$ = state$
+    ? (state$ as Observable<SearchSessionState>).pipe(
+        scan<SearchSessionState, boolean>((visible, s) => {
+          if (!visible) return s === SearchSessionState.Restored;
+          return STICKY_STATES.has(s);
+        }, false),
+        distinctUntilChanged()
+      )
+    : of(false);
+
+  const showCallout = useObservable(show$, false);
+
+  if (!isBackgroundSearchEnabled() || !showCallout) return null;
+
+  return (
+    <EuiPanel
```

Comment:
@elastic/kibana-design Can we get some feed back about adding a callout to unified search when using background search. This seems very heavy handed. The callout will push page contents down and wastes the most important part of the page with a large callout. Maybe unified search should just show an icon or something small to avoid pushing page content down?